### PR TITLE
Clarify how to install QuickLisp

### PR DESCRIPTION
**Pull request by objarni**
I had a hard time understanding how to properly install QuickLisp, tried clarifying the instructions

### CODE REVIEW

File: docs/INSTALLATION.md [hunk 1]
```diff
@@ -8,23 +8,23 @@ brew install sbcl
...
-and download
-[http://beta.quicklisp.org/quicklisp.lisp](http://beta.quicklisp.org/quicklisp.lisp).
+We will use [QuickLisp] for Lisp library management. To install QuickLisp, first download quicklisp.lisp: 
 
 ```bash
 curl -O http://beta.quicklisp.org/quicklisp.lisp
 ```
 
-Load it into your running Lisp implementation, and type:
+Then load it into your running Lisp implementation, e.g. using ```(load "quicklisp.lisp")``` in clisp. To complete the installation type:
```
Comment (wobh) [inline docs/INSTALLATION.md]: The implicit instructions are:

1. Run your lisp REPL.
2. From the REPL, load quicklisp installer, `(load "quicklisp.lisp")`.
3. Run installation `(quicklisp-quickstart:install)`.

**Comment by objarni** [comment]
Yes, but also the wording regarding quick lisp was confusing; it didn't mention what it was, and there was an 'and' in the sentence which made no sense.

I resolved by mentioning what QuickLisp is and moving the link beneath the instructions.

**Comment by yurrriq** [comment]
As a lisp user and Exercism contributor, I am definitely biased, but I'm struggling to see the value this PR adds.

**Comment by objarni** [comment]
I'm new to lisp, so I'm biased in another direction. I had no idea what QuickLisp was, and the instructions didn't explain nor give clear instructions on how to install. I tried to contribute to exercism by clarifying that after having struggled to understand what was what for an hour. 

I have done exercises in c++ and go track, and the instructions there where a lot clearer.

**Comment by yurrriq** [comment]
You raise a very valid concern, thank you, and obviously your experience is not ideal. Having read the original document for the first time today, it seems to me the idea is to "outsource" the description and installation instructions to quicklisp.org. I can understand how it could be confusing or daunting to a newcomer, but I'm hesitant to duplicate too much information.

The `curl` script copied here from QuickLisp seems self-explanatory enough to me, but this document seems to assume the reader knows how to load a Lisp file into a REPL. I believe there are some links to general introductory tutorials elsewhere on Exercism, and stick with the DRY mentality it feels a bit off to describe that again here. What do you think?

File: docs/INSTALLATION.md [hunk 2]
```diff
@@ -8,23 +8,23 @@ brew install sbcl
...
 
-(See their homepages for instructions for installing on other
-platforms.) 
+See their homepages for instructions for installing on other
+platforms.
 
-Then install [QuickLisp](http://www.quicklisp.org/beta/#installation)
-and download
-[http://beta.quicklisp.org/quicklisp.lisp](http://beta.quicklisp.org/quicklisp.lisp).
+We will use [QuickLisp] for Lisp library management. To install QuickLisp, first download quicklisp.lisp:
```
Comment (yurrriq) [inline docs/INSTALLATION.md]: Reading this again I can see how the "and" is confusing. Also, I think we should keep the  quicklisp.lisp link here.

File: docs/INSTALLATION.md (hunk 1 above)
Comment (yurrriq) [inline docs/INSTALLATION.md]: This seems resident. (See my other comment)

File: docs/INSTALLATION.md [hunk 3]
```diff
@@ -8,23 +8,23 @@ brew install sbcl
...
 
-Load it into your running Lisp implementation, and type:
+Then load it into your running Lisp implementation, e.g. using ```(load "quicklisp.lisp")``` in clisp. To complete the installation type:
 
 ```lisp
 (quicklisp-quickstart:install)
 ```
 
+Read more details about installing QuickLisp [here](http://www.quicklisp.org/beta/#installation).
+
```
Comment (yurrriq) [inline docs/INSTALLATION.md]: Assuming we keep this link above, this sentence feels unnecessary.

**Comment by yurrriq** [comment]
I've added some comments inline, but I'd love to get the opinion of another newcomer. Can anyone else chime in?

**Comment by objarni** [comment]
I expect the likelyhood of you getting feedback from some other newcomer to the Exercism Lisp track to be low in the "this week" time perspective. I hope I'm wrong though :)

**Comment by verdammelt** [comment]
I don't have time right now for a deeper reading of this (maybe tonight) but I wanted to point out that the instructions shouldn't say 'do this in clisp' but 'do this in your lisp' since the user may not be using clisp. We *suggest* either sbcl or clisp - but the user can use whatever Common Lisp implementation they'd like.

**Comment by objarni** [comment]
That's fine change it to lisp instead of clisp.

In general, please read both original page on exercism.io in-context, as well as changes in full, instead of as diffs. It's hard to get the reading flow when reading diffs.

**Comment by objarni** [comment]
Original page: http://exercism.io/languages/lisp/installing

**Comment by verdammelt** [comment]
@objarni first off - thanks! I think your change is really good and will make the documentation better. I think I wrote it and didn't think to put in some details you did which will help new people.

I like @wobh's comment above that there are specific steps: load REPL, load file in REPL, run install command, run add-to-init command.

Can we edit the file a little more to make those steps concrete? I'm suggesting something like:

```
(... curl command to get quicklisp)
Now launch your lisp implementation and paste in the following expressions and evaluate them:
> (quicklisp-quickstart:install)  ;; this will install quicklisp
> (ql:add-to-init-file)                 ;; this will add quicklisp setup to your init file (recommended)
```

(I think we can/should change the wording of using quick lisp to load `lisp-unit` but we can do that as a different PR.)

**Comment by objarni** [comment]
Thanks for feedback @verdammelt I have made some more changes following your ideas. I believe you can read the result page by following this link (diffs' are really terrible at this documentation, aren't they!?)

https://github.com/objarni/xlisp/blob/d56fabf18c6de653bf87f1b82ddc7ed377dad955/docs/INSTALLATION.md

**Comment by wobh** [comment]
Looks good to me.

**Comment by verdammelt** [comment]
Looks good! Thanks for the updated @objarni.

Merging!

**Pull request merged**